src/.dockerignore

```diff
@@ -0,0 +1,5 @@
+**/bin
```

Comment:
We need the additional `.dockerignore` because the docker-compose context used for the dacpac.deploy build is the `src` directory. Without the additional `.dockerignore` the docker-compose build would fail.

I think the only way to avoid this would be to see if it is possible to use the solution directory as the build context. I can look into it but I'm thinking it might be one for tech debt...

Reply:
This is the pattern used by the other back end projects e.g. Identity.Api, Ordering.Api etc..
Example taken from the docker-compose file in Ordering.Api:

```
    build: 
      context: .
      dockerfile: src/NHSD.BuyingCatalogue.Ordering.Api/Dockerfile
```

Does mean in the Docker file the directory paths will change with a prefix of 'src/', but it keeps it simple as it follows the project directory paths.

If the docker image required any additional files it is in the right context to navigate to any part of the project structure. 

Also I think this will reduce the confusion of having another docker ignore file.

Reply:
I've refactored to use the solution root as the build context although I'm not entirely convinced that it is an improvement or any less confusing. 😀